CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+1.1.0 / 2021/03/XX
+====================
+- Add `sessionPool.addSession()` `async` function to add a new session to the session pool (possibly with the provided options, e.g. with specific session id). Add the ability to retrieve a session from the session pool with the specific session id.
```

Comment:
> Add the ability to retrieve a session from the session pool with the specific session id.

This would deserve a second bullet point with reference to the `getSession` function.

Reply:
Was not sure how detailed it should be (either each update is a separate bullet point or everything (adding/getting) around session by id is one entry). Updated